Examples/Algorithms/Generators/ActsExamples/Generators/VertexGenerators.hpp

```diff
@@ -46,4 +46,40 @@ struct GaussianPrimaryVertexPositionGenerator
   }
 };
 
+//
+struct GaussianDisplacedVertexPositionGenerator
+    : public EventGenerator::PrimaryVertexPositionGenerator {
+  Acts::Vector4 stddev = {0.0, 0.0, 0.0, 0.0};
+  Acts::Vector4 mean = {0.0, 0.0, 0.0, 0.0};
+
+  Acts::Vector4 operator()(RandomEngine& rng) const override {
+    double min_value = -2 * M_PI;  // -π
+    double max_value = 2 * M_PI;   // π
+
+    // Create a uniform real distribution object with range [-π, π)
+    std::uniform_real_distribution<> uniform(min_value, max_value);
+
+    // Create a normal distribution object with mean 0.0 and standard
+    // deviation 1.0
+    auto normal = std::normal_distribution<double>(0.0, 1.0);
+    Acts::Vector4 rnd = {normal(rng), uniform(rng), normal(rng), normal(rng)};
+
+    // Compute the cylindrical coordinates with deviations applied
+    Acts::Vector4 cylindrical = mean + rnd.cwiseProduct(stddev);
+
+    // Extract coordinates from the vector
+    double r = cylindrical(0);
+    double phi = cylindrical(1);
```

Comment:
I believe here you should be able to just do 
```suggestion
    double phi = uniform(rng);
```

Overall, using the vector of means and standard deviations makes more sense for the original vertex position generator. I'd suggest to just give named variables like:

```cpp
double rMean = 0;
double rStdDev = 0;
double zMean = 0;
double zStdDev = 0;
double tMean = 0;
double tStdDev = 0;
```

which might be less confusing than the vector members.